app/customer.py

```diff
@@ -0,0 +1,37 @@
+from math import dist
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
+                 money: float,
+                 car_consumption: float) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.home = location
+        self.location = location
+        self.money = money
+        self.shop = "Please approve task"
+        self.home = location
+        self.car_consumption = car_consumption
+
+    def customer_info(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def shop_visit(self, shop_name: str, cost: float) -> None:
+        print(f"{self.name}'s trip to the {shop_name} costs {cost}")
+
+    def change_location(self, destination: list) -> None:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]